Directory.Build.props

```diff
@@ -22,11 +22,14 @@
     <IsNetStandard Condition="$(NetStandardVersions.Contains('$(TargetFramework)'))">true</IsNetStandard>
     <IsNetCore>false</IsNetCore>
     <IsNetCore Condition="$(NetCoreVersions.Contains('$(TargetFramework)'))">true</IsNetCore>
+    <IsNetFramework>false</IsNetFramework>
+    <IsNetFramework Condition="$(NetFrameworkVersions.Contains('$(TargetFramework)'))">true</IsNetFramework>
```

Comment:
These two lines seem to contradict one another

Reply:
First line will provide a default value (`false`) for a variable.
Second line will override it with `true` if the condition is met